fedmsg_meta_umb/tests/test_rpmdeplint.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# fedmsg_meta_umb is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# fedmsg_meta_umb is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with fedmsg; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Authors: Gowrishankar Rajaiyan <grajaiya@redhat.com>
+
+import fedmsg.tests.test_meta
+
+
+class TestRPMDEPLINTStart(fedmsg.tests.test_meta.Base):
+    """ `RPMDEPLINT <https://rpmdeplint.readthedocs.io/en/latest/`_ finds errors in RPM packages in the context
```

Comment:
It's not an acronym, I suggest just spelling it "rpmdeplint".

    The `rpmdeplint <https://pagure.io/rpmdeplint>`_ tool finds errors...

Also you have a Sphinx syntax error here (missing `>`).